tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -591,10 +591,11 @@ tensorflow::Status CreateTRTNode(const std::vector<EngineInfo>& infos, int pos,
     }
     VLOG(1) << ins;
   }
-  node_builder.Input(inputs);
+  // Add control inputs first, just in case.
```

Comment:
Would you explain why doing this and add it to the comment (i.e. `just in case <what>`)?